scalableminds/webknossos
# Add "hide/show other trees" to tree context-menu

There already is a functionality in webknossos to "Hide all inactive trees", i.e. toggle the visibility of all other trees but the currently selected one. This feature is bound to keyboard hotkey `2`. 
In this PR, I exposed this functionality through the right-click context menu for each tree/group.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create a skeleton annotation with a few trees and groups.
- Right-click on a tree/group and select "Hide/Show all other trees"
- The selected tree/group should now: 
  - be selected (blue background behind name)
  - the visibility of all tree/groups should be toggled
 - Right-click again to undo

### Issues:
- no issue 

------
(Please delete unneeded items, merge only when none are left open)
